frontend/package.json

```diff
@@ -10,13 +10,19 @@
   },
   "dependencies": {
     "expo": "^35.0.0",
+    "expo-constants": "~7.0.0",
     "native-base": "^2.13.8",
     "prop-types": "^15.7.2",
     "react": "16.8.3",
     "react-dom": "16.8.3",
     "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
+    "react-native-gesture-handler": "~1.3.0",
+    "react-native-reanimated": "~1.2.0",
```

Comment:
where do you use it? what is this for?

Reply:
please don't delete immediately, I'm just not sure what is this for, I don't see this being used anywhere